script/Arcueid/Day 5/05_00_ARC05_5.txt

```diff
@@ -707,7 +707,7 @@ The unraveled chains and the raised anchors strike the surface of the earth.
 [sha:f39ef2ae76b3c855e00ab4ff809b18f7565d1457]{
 -- Page 420, Offset 9698.
 -- 眼下には火の粉を散らす石の<楼閣|ろうかく>。
-Below her is a tall stone building scattering sparks of fire.
+Below her is a tall, stone building scattering sparks of fire.
```

Comment:
It's to separate the adjectives.

Reply:
Doesn't work here. We do use this pattern elsewhere, but if I were to attempt to define the rule it would be for adjectives that are in the same vein. These two aren't really.

Reply:
this is just me speculating, but I think "stone building" is essentially one noun in english, so we wouldn't use an adjective to separate stone from tall because they're not both adjectives.

Reply:
heh works for me